src/actions/meetings/updateMeeting.mjs

```diff
@@ -0,0 +1,38 @@
+import moment from 'moment-timezone';
+
+
+const updateMeeting = async (values, id) => {
+    let attendees = values.attendees.split(",")
+    for (let idx in attendees) {
+        attendees[idx] = attendees[idx].trim()
+    }
+
+    const body = {
+        name: values.name.trim(),
+        organizerName: "John", // TODO: set
```

Comment:
Could you explain why there are TODOs here? I'm assuming you want to automate the input?

Reply:
Yep. Once we have our user auth set up, this will automatically be set.